gtk4/src/expression.rs

```diff
@@ -182,6 +182,114 @@ impl glib::value::ToValueOptional for Expression {
     }
 }
 
+// rustdoc-stripper-ignore-next
+/// Trait containing convenience methods in creating [`gtk::PropertyExpression`] that
+/// looks up a property of a [`glib::Object`].
+///
+/// # Example
+///
+/// `label_expression` is a [`gtk::Expression`] that looks up at Button's `label`
+/// property.
+///
+/// ```no_run
+/// use gtk4::prelude::*;
```

Comment:
```suggestion
/// use gtk::prelude::*;
```

We typically expect that people rename it to `gtk`

Reply:
I think that will make the tests fail

Reply:
It won't, if you add a little hidden comment (see gtk4-macros) that does `use gtk4 as gtk;`